web/app/js/components/Namespaces.jsx

```diff
@@ -0,0 +1,127 @@
+import _ from 'lodash';
+import CallToAction from './CallToAction.jsx';
+import ConduitSpinner from "./ConduitSpinner.jsx";
+import ErrorBanner from './ErrorBanner.jsx';
+import MetricsTable from './MetricsTable.jsx';
+import PageHeader from './PageHeader.jsx';
+import { processRollupMetrics } from './util/MetricUtils.js';
+import React from 'react';
+import './../../css/list.css';
+import 'whatwg-fetch';
+
+export default class Namespaces extends React.Component {
+  constructor(props) {
+    super(props);
+    this.api = this.props.api;
+    this.handleApiError = this.handleApiError.bind(this);
+    this.loadFromServer = this.loadFromServer.bind(this);
+    this.state = this.getInitialState(this.props.location);
+  }
+
+  getInitialState(location) {
+    location = location || {};
+    let urlParams = new URLSearchParams(location.search);
+    let ns = urlParams.get("ns") || "default";
```

Comment:
is there a way to force navigation to `/namespaces?ns=default`?
or, to put it another way, when i (manually) navigated to `/namespaces` without request params, i was expecting to see a list of namespaces, with links to each namespace page. related: should that namespaces page exist? just another type of `ResourceList`?

Reply:
Yep you're right; at first I thought that ServiceMesh would be the only index into namespaces, but it's quite easy to just add another page that lists stats by Namespace (just another ResourceList, as you mentioned!). So I've added that.

Reply:
![screen shot 2018-05-08 at 12 40 03 pm](https://user-images.githubusercontent.com/549258/39779519-4ce60856-52be-11e8-9d0b-8c228f0affa9.png)